CODEOWNERS

```
@@ -65,6 +65,7 @@ go.mod @fleetdm/go
 # Handbook
 /handbook/company @mikermcneil
 /handbook/business-operations @hollidayn
+/handbook/business-operations/ceo-handbook @charlottechance
```

```suggestion
/handbook/business-operations/ceo-handbook.md @charlottechance
```

almost there! a note that due how this [file works](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax), with this change only you (Charlie) will get notifications about changes in `ceo-handbook.md`.

If instead you want both you and Nathan to get notifications for this file then you want to do:

```suggestion
/handbook/business-operations/ceo-handbook.md @charlottechance @hollidayn
```

---

Hey @roperzh, thanks for the suggestions. That makes sense. In this case I think we'll want to keep one DRI per page. 
Rationale: one DRI per page is our standard across the company. Nathan owns BizOps and I own anything CEO-related. Thanks for the review!